Firebase/Database/Utilities/FUtilities.h

```diff
@@ -41,42 +39,26 @@
 
 @end
 
-typedef enum {
-    FLogLevelDebug = 1,
-    FLogLevelInfo = 2,
-    FLogLevelWarn = 3,
-    FLogLevelError = 4,
-    FLogLevelNone = 5
-} FLogLevel;
-
 // Log tags
 FOUNDATION_EXPORT NSString *const kFPersistenceLogTag;
 
 #define FFLog(code, format, ...) FFDebug((code), (format), ##__VA_ARGS__)
 
 #define FFDebug(code, format, ...)                                             \
     do {                                                                       \
-        if (FFIsLoggingEnabled(FLogLevelDebug)) {                              \
-            FIRLogDebug(kFIRLoggerDatabase, (code), (format), ##__VA_ARGS__);  \
-        }                                                                      \
+        FIRLogDebug(kFIRLoggerDatabase, (code), (format), ##__VA_ARGS__);      \
     } while (0)
 
 #define FFInfo(code, format, ...)                                              \
     do {                                                                       \
-        if (FFIsLoggingEnabled(FLogLevelInfo)) {                               \
-            FIRLogError(kFIRLoggerDatabase, (code), (format), ##__VA_ARGS__);  \
-        }                                                                      \
+        FIRLogError(kFIRLoggerDatabase, (code), (format), ##__VA_ARGS__);      \
```

Comment:
Presumably this should be `FIRLogInfo` instead of `FIRLogError` - I realize it was just a copy paste from before but that could also be why Database is particularly chatty during logs.

Reply:
Or, maybe since there's no `FFError`, we should rename it as such.

Reply:
Good catch! I'll update it to use proper FIRLogger level if we decide to move ahead with the change.

Reply:
This seems like a bug more than a conscious decision. I can't come up with a reason why we would log INFOs at a higher level than WARN.

Can you update to use the proper level here? Please also add a changelog.

Reply:
I guess,  the intention was to make Database logging level to suppress the global logging level (if global level if error but DB level is Info, then DB logs are printed anyway). Doesn't look really readable though. @schmidt-sebastian Do you still think we will need to change it even if we decide to postpone the public API change? See also discussion in go/firebase-database-ios-log-level-api.

Reply:
I can't get myself to recommend that you keep it as is, so I would suggest changing it. Info logs shouldn't log at a higher level than warning logs. Let's just make sure to make it very obvious in the release notes.

Reply:
If my search was right there’s only a single log that uses this function so it doesn’t have a super high impact either way.